en/docs/develop/restapis/association.yaml

```diff
@@ -148,49 +153,293 @@ paths:
-          name : SwitchUserReqeust
-          in : body
+        - name: associated-user-id
+          in: path
           required: true
+          type: string
+      summary: Delete given user account from my user associations
+      description: |
+        This API is used to delete the given user account from the associations of the auhtentiated user.
+
+          <b>Permission required:</b>
+
+          * /permission/admin/login
+      responses:
+        204:
+          description: No content
+        401:
+          description: Unauthorized
+        403:
+          description: Resource Forbidden
+        500:
+          description: Server Error
           schema:
-            $ref: '#/definitions/AssociationSwitchRequest'
+            $ref: '#/definitions/Error'
+      tags:
+        - me
 
-      summary: Switch user account
+  /me/federated-associations:
+    get:
+      tags:
+        - me
+      summary: Retrive the federated associations of the authenticated user.
       description: |
-          This API is used to switch the user account in the user portal. This API is not implemented yet.
-                  <b>Permission required:</b>
-          * /permission/admin/manage/identity/user/association/update
+        This API is used to retrieve the federated associations of the authenticated user.
+
+          <b>Permission required:</b>
+
+          * /permission/admin/login
       responses:
         200:
-          description: OK
-        400:
-          description: Bad Request
+          description: Successful operation
+          schema:
+            type: 'array'
+            items:
+              $ref: '#/definitions/FederatedAssociation'
+        204:
+          description: No content
+        401:
+          description: Unauthorized
+        403:
+          description: Resource Forbidden
+        500:
+          description: Server Error
           schema:
-             $ref: '#/definitions/Error'
+            $ref: '#/definitions/Error'
+    delete:
+      summary: Delete all my federated user associations
+      description: |
+        This API is used to delete all federated associations of the auhtentiated user.
+
+          <b>Permission required:</b>
+
+          * /permission/admin/login
+      responses:
+        204:
+          description: No content
         401:
           description: Unauthorized
         403:
           description: Resource Forbidden
         500:
           description: Server Error
           schema:
-             $ref: '#/definitions/Error'
-        501:
-          description: Not Implemented
+            $ref: '#/definitions/Error'
+      tags:
+        - me
+
+  /me/federated-associations/{id}:
+    delete:
+      parameters:
+        - name: id
+          in: path
+          required: true
+          type: string
+      summary: Delete my federated association
+      description: |
+        This API is used to delete a federated association of the auhtentiated user.
```

Comment:
authenticated

Reply:
Addressed in 861bcaec1512c2d43091b40273eaf2c2dce11f3e